security/security-design/shared-assets/email-security-dns-check/README.md

```diff
@@ -0,0 +1,59 @@
+## Purpose of the email security DNS check
```

Comment:
Ad a disclaimer for http://ip-api.com usage conditions!

Reply:
Added the disclaimer.